src/components/button/index.md.njk

```diff
@@ -47,7 +47,7 @@ Avoid using multiple default buttons on a single page. Having more than one main
 ### Start buttons
 
 Use a start button for the main call to action on your service’s [start page](/patterns/start-using-a-service/).
-Start buttons do not submit form data, so they use a link tag rather than a button tag.
+Start buttons don't usually submit form data, so they use a link tag rather than a button tag.
```

Comment:
Minor suggested change to:
- match GOV.UK style to spell out negative contractions
- avoid the suggesting that Start buttons have a mind of their own

```suggestion
Start buttons do not usually submit form data, so use a link tag instead of a button tag.
```